packages/shadydom/src/patch-events.js

```diff
@@ -44,7 +44,10 @@ const supportsEventOptions = (() => {
 
 const parseEventOptions = (optionsOrCapture) => {
   let capture, once, passive, shadyTarget;
-  if (optionsOrCapture && optionsOrCapture instanceof Object) {
+  if (
+    optionsOrCapture &&
+    (typeof optionsOrCapture === 'object' || optionsOrCapture instanceof Object)
```

Comment:
The only values I know of that are not `typeof o === 'object'` but are `o instanceof Object` are functions. `typeof o === 'function'` should always be faster.

Reply:
Done.